src/app/components/contactUs/ContactUsForm.tsx

```diff
@@ -67,120 +64,109 @@ const ContactUsForm: React.FC = () => {
   };
 
   return (
-    <LoaderWrapper isLoading={isLoading} expandLoaderWidth>
-      <div className={styles.container}>
-        <div className={styles.title}>
-          <Typography
-            variant={TypographyVariant.H2}
-            className={styles.mobileTitle}
+    <div className={styles.container}>
+      <div className='row'>
+        <ContactUsLeftBanner />
+        <div className={classNames(styles.formColumn, 'col-md-8')}>
+          <LoaderWrapper
+            isLoading={isLoading}
+            className={styles.formColumnWrapper}
+            expandLoaderWidth
           >
-            Contact Us
-          </Typography>
-          {submissionSuccess ? (
-            <Typography variant={TypographyVariant.Body1}>
-              Thank you for contacting us. We will get back to you shortly.
-            </Typography>
-          ) : (
-            <Typography variant={TypographyVariant.Body1}>
-              We will try our best to get back to you as soon as possible
-            </Typography>
-          )}
-        </div>
-        <Form methods={methods} onSubmit={methods.handleSubmit(onSubmit)}>
-          <div className={styles.textboxBody}>
-            <div className={styles.nameArea}>
-              <div>
-                <TextBox
-                  name='firstname'
-                  label='First Name'
-                  required
-                  placeholder='Enter your first name'
-                  pattern={NAME_REGEX}
-                  showLabel
-                />
-              </div>
-              <div>
-                <TextBox
-                  name='lastname'
-                  label='Last Name'
-                  required
-                  placeholder='Enter your last name'
-                  pattern={NAME_REGEX}
-                  showLabel
-                />
-              </div>
-            </div>
-            <div className={styles.phoneArea}>
-              <div>
-                <TextBox
-                  name='phone'
-                  label='Phone number'
-                  placeholder='Enter your phone number'
-                  pattern={PHONE_REGEX}
-                  showLabel
-                />
+            <Form methods={methods} onSubmit={methods.handleSubmit(onSubmit)}>
+              <div className={styles.textboxBody}>
+                <div className={styles.nameArea}>
+                  <div>
+                    <TextBox
+                      name='firstname'
```

Comment:
Check how the form is rendered in the auth pages. We don't need this amount of nesting - (div.textboxBody > div.nameArea > div). We probably don't need all this classes either. Remove the use of `gap` css property, let's stick with col={6} for 'First Name' & 'Last Name'. The total padding of 30px in between them should be fine.

Reply:
Provided an update. Let me know if you have a suggestion.